packs/pathfinder-monster-core/animated-armor.json

```diff
@@ -211,56 +211,7 @@
                 "temp": 0,
                 "value": 20
             },
-            "immunities": [
-                {
-                    "type": "bleed"
-                },
-                {
-                    "type": "death effects"
-                },
-                {
-                    "type": "disease"
-                },
-                {
-                    "type": "doomed"
-                },
-                {
-                    "type": "drained"
-                },
-                {
-                    "type": "fatigued"
-                },
-                {
-                    "type": "healing"
-                },
-                {
-                    "type": "mental"
-                },
-                {
-                    "type": "nonlethal attacks"
-                },
-                {
-                    "type": "paralyzed"
-                },
-                {
-                    "type": "poison"
-                },
-                {
-                    "type": "sickened"
-                },
-                {
-                    "type": "spirit"
-                },
-                {
-                    "type": "unconscious"
-                },
-                {
-                    "type": "vitality"
-                },
-                {
-                    "type": "void"
-                }
-            ],
+            "immunities": [],
```

Comment:
These are implicit due to being a construct. Additionally, `"death effects"` and `"nonlethal attacks"` weren't sluggified.